plugins/portals-list.js

```diff
@@ -14,8 +14,22 @@ window.plugin.portalslist.sortOrder = -1;
 window.plugin.portalslist.enlP = 0;
 window.plugin.portalslist.resP = 0;
 window.plugin.portalslist.neuP = 0;
+window.plugin.portalslist.visitedP = 0;
+window.plugin.portalslist.capturedP = 0;
+window.plugin.portalslist.scoutControlledP = 0;
+
 window.plugin.portalslist.filter = 0;
 
+window.plugin.portalslist.historyCodePoints =[];
+[0x026AA, // white dot
+ 0x1F7E1, // yellow dot
+ 0x02B55, // red empty dot (should never show)
+ 0x1F534, // red dot
+ 0x1F7E3  // violet dot
+ ].forEach (function (value) {
+   window.plugin.portalslist.historyCodePoints.push(String.fromCodePoint(value))
+});
```

Comment:
```suggestion
window.plugin.portalslist.historyCodePoints = [
  0x026AA, // white dot
  0x1F7E1, // yellow dot
  0x02B55, // red empty dot (should never show)
  0x1F534, // red dot
  0x1F7E3  // violet dot
].map(String.fromCodePoint);
```

Reply:
But I do not like how that is then handled.
All these `window.plugin.portalslist.historyCodePoints[value]` occurrences are ugly.

Reply:
Also I feel that those functions are redundant entities:
`window.plugin.portalslist.visitedValue`
`window.plugin.portalslist.scoutControlledValue`

Just hating spaghetti-code.

Reply:
But it is the only way to expose them. 
We can still use a secondary pointer in the code, but I would leave that to other PR.

Reply:
> But it is the only way to expose them.

Why we need to expose them?

Reply:
because we wanted users to be able to change the way it is displayed.
Or an other plugin to change the appearance.